src/main/java/uk/ac/ebi/ena/readtools/validator/ReadsValidator.java

```diff
@@ -139,21 +139,17 @@ public class ReadsValidator
     {
         try
         {
-            if( files.size() > 2 )
-            {
-                result.add(ValidationMessage.error("More than two fastq files were provided: " + files.size()));
-                return;
-            }
+            int readLimit = manifest.isQuick() ? QUICK_READ_LIMIT : EXTENDED_READ_LIMIT;
 
-            FastqScanner fs = new FastqScanner(
-                    manifest.isQuick() ? DEFAULT_QUICK_RUN_DURATION : null, manifest.getPairingHorizon() ) {
+            //Higher expected size results in better accuracy of validation techniques used by FastqScanner.
+            int expectedSize = readLimit * 5;
```

Comment:
expectedSize is bloom filter specific configuration parameter. Optimally it would not be visible to clients but would be set based from the readLimit when we create the bloomfilter internally.